src/Cronofy/Conferencing.cs

```diff
@@ -0,0 +1,78 @@
+namespace Cronofy
+{
+    /// <summary>
+    /// Class for the deserialization of the options for a read event
+    /// response.
+    /// </summary>
+    public sealed class Conferencing
+    {
+        /// <summary>
+        /// Gets or sets the conferencing provider name.
+        /// </summary>
+        /// <value>
+        /// The conferencing provider name.
+        /// </value>
+        public string ProviderName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the conferencing URL of the event.
+        /// </summary>
+        /// <value>
+        /// The conferencing URL of the event.
+        /// </value>
+        public string JoinUrl { get; set; }
+
+        /// <inheritdoc/>
+        public override bool Equals(object obj)
+        {
+            if (ReferenceEquals(null, obj))
```

Comment:
1/10

This is quite a complicated and longwinded way to implement `Equals()` typically I would do the following:

```
if (obj is Conferencing conferencingObj) 
{
    return Equals(conferencingObj);
}

return false;
```

This will gracefully handle `null` for you, though you may want to re-add the `ReferenceEquals(this, obj)` if performance is a concern.